src/app/users/users.component.html

```diff
@@ -83,6 +84,8 @@ <h1 i18n>All Users</h1>
           <button mat-icon-button *ngIf="element.roles.length > 0" (click)="removeRole(element)">
             <mat-icon>lock_open</mat-icon>
           </button>
+          <button mat-raised-button color="primary"  *ngIf="!element.myTeamInfo" (click)="addTeams([{'_id': element._id}])">Add Team</button>
+          <button mat-raised-button color="primary"  *ngIf="element.myTeamInfo" (click)="removeTeam(element._id)">Remove Team</button>

```

Comment:
we can use it as in else part using ng-container. Could you please check it once ?